spec/controllers/catalog_controller_spec.rb

```diff
@@ -137,6 +137,14 @@
         put :search, query: 'query'
         expect(assigns(:equipment_model_results)).to eq([@equipment_model])
       end
+      it 'should give unique results even with multiple matches' do
+        @equipment_model = FactoryGirl.create(:equipment_model,
+                                              active: true,
+                                              name: 'query',
+                                              description: 'query')
+        put :search, query: 'query'
+        expect(assigns(:equipment_model_results)).to eq([@equipment_model])
```

Comment:
Sure, I guess it's implied in the one element array, but more clarity can't hurt.